.github/workflows/deploy-tag.yml

```diff
@@ -0,0 +1,76 @@
+name: Deploy Tagged Containers
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  tags:
+    outputs:
+      major: ${{ steps.tag.outputs.major }}
+      minor: ${{ steps.tag.outputs.minor }}
+      patch: ${{ steps.tag.outputs.patch }}
+    steps:
+    - id: tag
+      run: |
+        MAJOR_VERSION=$(echo $GITHUB_REF | sed -rn 's#^/refs/tags/(v3)\.([0-9]+)\.([0-9]+)$#\1#p')
+        echo "::set-output name=major::${MAJOR_VERSION}"
+        MINOR_VERSION=$(echo $GITHUB_REF | sed -rn 's#^/refs/tags/(v3)\.([0-9]+)\.([0-9]+)$#\1.\2#p')
+        echo "::set-output name=minor::${MINOR_VERSION}"
+        PATCH_VERSION=$(echo $GITHUB_REF | sed -rn 's#^/refs/tags/(v3)\.([0-9]+)\.([0-9]+)$#\1.\2.\3#p')
+        echo "::set-output name=patch::${PATCH_VERSION}"
+  deploy-github-containers:
+    needs: tags
+    name: Github Registry tag
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - php-apache
+          - nginx
+          - fpm
+          - fpm-dev
+          - admin
+          - migrate-database
+          - update-frontend
+          - consume-messages
+          - ilios-mysql
+          - ilios-mysql-demo
```

Comment:
These should definitely be prefixed with `ilios-`, correct?  Just making sure.

Reply:
Thanks!  Fixed.